lodash.src.js

```diff
@@ -11692,9 +11702,10 @@
           if (onlyLazy) {
             value = value.clone();
             value.__actions__.push(action);
-            return func.call(value);
+            return func.apply(value, args);
           }
-          return lodashFunc.call(undefined, this.value())[0];
+          var result = this.thru(interceptor).value();
+          return isTaker ? result[0] : result;
```

Comment:
Thoughts on the approach in https://github.com/lodash/lodash/commit/5e9ca3fe9a3946b0858b0481e94a65e2011c66f4 vs this one?